app/src/main/java/com/keylesspalace/tusky/fragment/NotificationsFragment.java

```diff
@@ -613,17 +633,29 @@ private void insert(List<Notification> newNotifications, int pos) {
             return;
         }
 
-        if(newNotifications.size() >= LOAD_AT_ONCE) {
-            Notification placeholder = new Notification();
-            placeholder.type = Notification.Type.PLACEHOLDER;
-            newNotifications.add(placeholder);
+        List<Either<Placeholder, Notification>> lifedNew = liftNotificationList(newNotifications);
+
+        if (newNotifications.size() >= LOAD_AT_ONCE) {
```

Comment:
@connyduck could you explain this one please? I don't understand this condition, if we loaded more than the limit, then we add the placeholder. How can we load more than the limit? I'm lost.